kensu/pandas/extractor.py

```diff
@@ -70,12 +70,15 @@ def extract_stats(self, df):
             #Extract datetime for timeliness
             date_df = df.select_dtypes(['datetime', 'datetimetz'])
             date_dict = {}
-            date_describe = date_df.describe().to_dict()
-            for col in date_describe:
-                first = date_describe[col]['first'].timestamp()*1000
-                last = date_describe[col]['last'].timestamp()*1000
-                date_dict[col + '.first'] = first
-                date_dict[col + '.last'] = last
+            # if not checked, when there's no Datetime types,
+            # it would throw `ValueError: Cannot describe a DataFrame without columns`
+            if not (date_df.ndim == 2 and date_df.columns.size == 0):
```

Comment:
P.S. the condition taken as-is from pandas code in the error stacktrace, just added `not`.